components/About.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <div>
+    <section id="about" class="contact page intermediate-section full-height parallax about-us-desktop" @mouseover="changeUrl('about')">
+      <div class="overlay">
+        <div class="container valign">
+          <div class="row">
+            <article class="col-md-12 col-lg-8 col-lg-offset-2 main-caps text-center">
+              <h3 class="white">
+                <span>{{ about.title }}</span>
+              </h3>
+              <div class="liner color-bg" />
+              <p class="white">
+                {{ about.desktopSection1 }}
+              </p>
+              <p class="white">
+                {{ about.desktopSection2 }}
+              </p>
+              <p class="white">
+                {{ about.desktopSection3 }}
+              </p>
+              <p class="white">
+                {{ about.desktopSection4 }}
+              </p>
```

Comment:
Can we have loop instead of hard code 4 section?

Reply:
Updated to for loops